### DISCUSSION TITLE
Add back the feature to mod PIN(s) (Unpatch duped 3Pin/2pin Mod)

### DISCUSSION THREAD

**Discussion by ggobzzzjsjssjsj** [LBPUnion/ProjectLighthouse#1024] [Ideas]
Hello, many of us modders, whom been modding since basically the start of LBP Modding, we are quite dissapointed by the fact that "modding" pins is categorized under "cheating scores" how illogical can that be? 

(There hasn't ever been a compain about PIN Modding, even Lighthouse has a level for pin(s) modding)
Source: https://beacon.lbpunion.com/slot/11879/v20-pin-giver-tool-giveaway 

I get why people are upset with cheating scores, but no one considers PIN(s) as an big thing or even an achivenment anymore, many pins at this moment aren't even legitmately unlockable because of the Official servers LBP2/3 (PS3) that got shutdown back in 2021. Which made PIN(s) worthless by a whole because you can't even unlock most of the secret pin(s) anymore.  

If we shouldn’t have duplicates of the same pin, then we could argue that having retexured shirts, gmat swapped clothing pieces and custom models should essentially also be categorized under "cheating scores". Since it’s something that you’re not supposed to be having in the game and that should logically also be considered as "cheating"

The reason why most modders want it back, it's because it makes your profile look clean and nice with having three of the same pins, where on the other hand having to different pins, doesn't necessarily do the trick to make your profile look good. 

I don't see why "modding" pin(s) is harmful to the servers, and how it ruins other people's experience with the server? How can Beacon allow basically the same thing, but not the other one? 

![librewolf_vxlXBKzJZe](https://github.com/LBPUnion/ProjectLighthouse/assets/100964481/75c4bbe8-fee2-41e0-9941-52e910334ff2)

**Comment by sudokoko** [comment]
Duplicate pins were disabled across the scope of the project due to abuse by bad actors.

> "modding" pins is categorized under "cheating scores"

This is not true. However, I see your point here.

I would not be opposed to implementing the ability for the patch to be enabled/disabled via configuration option, as there's no point in removing the patch altogether. In terms of Beacon, we can look into re-enabling the functionality.

**Reply by ggobzzzjsjssjsj** [reply]
"This is not true. However, I see your point here."  I was informed by an USC Director, that "modding pins" does fall under the category "cheating scores", now i'm confused of what it falls under. 
Source: [Discord_UyEJdjtWDl](https://github.com/LBPUnion/ProjectLighthouse/assets/100964481/ce209bc4-e9ab-4c61-86fd-15b7e26f1a6e)

And how did bad people manage to abuse others by duplicating pins, i don't get it really. How it would affect others by showcasing  modded pins. Is it just the people who had the modded duped pin(s) that were generally some a-holes or something else?

Thanks for responding, and especially very kind of you to look into "re-enabling the functionality"

**Reply by sudokoko** [reply]
>  I was informed by an USC Director, that "modding pins" does fall under the category "cheating scores", now i'm confused of what it falls under.

It does not fall under cheated scores. I'm apart of the Beacon moderation team, I can assure you it's not.

> Is it just the people who had the modded duped pin(s) that were generally some a-holes

While they weren't particularly nice people, our thought process was that it wasn't intended game functionality, so we prevented it.

No problem, I will keep you updated on this.

**Reply by ggobzzzjsjssjsj** [reply]
Alright thanks!

**Reply by ggobzzzjsjssjsj** [reply]
Any updates?
